crypto/merkle/simple_tree.go

```diff
@@ -51,8 +57,22 @@ func simpleHashFromHashes(hashes [][]byte) []byte {
 	case 1:
 		return hashes[0]
 	default:
-		left := simpleHashFromHashes(hashes[:(len(hashes)+1)/2])
-		right := simpleHashFromHashes(hashes[(len(hashes)+1)/2:])
+		k := getSplitPoint(len(hashes))
+		left := simpleHashFromHashes(hashes[:k])
+		right := simpleHashFromHashes(hashes[k:])
 		return SimpleHashFromTwoHashes(left, right)
 	}
 }
+
+func getSplitPoint(length int) int {
```

Comment:
Can we add a comment that this computes `k` from the rfc? (As in https://tools.ietf.org/html/rfc6962#section-2.1)